src/livestreamer/plugins/bliptv.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from livestreamer.plugin import Plugin, PluginError
+from livestreamer.plugin.api import http
+from livestreamer.stream import HTTPStream
+
+_url_re = re.compile("(http(s)?://)?blip.tv/.*-(?P<videoid>\d+)")
+VIDEO_GET_URL = 'http://player.blip.tv/file/get/{0}'
+SINGLE_VIDEO_URL = '.*\.((mp4)|(mov)|(m4v)|(flv))'
+
+
+def get_quality_dict(quality_list):
+    quality_list.sort()
+    quality_dict = {}
+    i = 0
+    for bitrate in quality_list:
```

Comment:
You can use `for i, bitrate in enumerate(quality_list)` here instead to avoid incrementing manually.